src/parser/hunter/survival/modules/talents/TipOfTheSpear.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
+import ItemDamageDone from 'interface/others/ItemDamageDone';
+import { RAPTOR_MONGOOSE_VARIANTS, TIP_DAMAGE_INCREASE, TIP_MAX_STACKS } from 'parser/hunter/survival/constants';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+
+/**
+ * Kill Command increases the damage of your next Raptor Strike by 20%, stacking up to 3 times.
+ *
+ * Example log: https://www.warcraftlogs.com/reports/BjmyHd9zt8RYJrWA/#fight=3&source=1
+ */
+
+const MS_BUFFER = 100;
+
+class TipOfTheSpear extends Analyzer {
+
+  spenderCasts = 0;
+  stacks = 0;
+  usedStacks = 0;
+  wastedStacks = 0;
+  damage = 0;
+  lastApplicationTimestamp = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.TIP_OF_THE_SPEAR_TALENT.id);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.KILL_COMMAND_CAST_SV.id && !RAPTOR_MONGOOSE_VARIANTS.includes(spellId)) {
+      return;
+    }
+
+    if (spellId === SPELLS.KILL_COMMAND_CAST_SV.id) {
+      if (this.stacks === TIP_MAX_STACKS && event.timestamp > this.lastApplicationTimestamp + MS_BUFFER) {
```

Comment:
Is the timestamp necessary at all? If you cast KC at max stacks, you wasted the stack regardless of the time, no? Also, does it mean the cast happens *after* applybuff/applybuffstack?